erpnext/accounts/doctype/fixed_asset_account/depreciation_report.py

```diff
@@ -0,0 +1,154 @@
+# Copyright (c) 2013, Web Notes Technologies Pvt. Ltd. and Contributors and contributors
+# For license information, please see license.txt
+
+from __future__ import unicode_literals
+from __future__ import division
+import frappe
+from datetime import date, timedelta
+from datetime import datetime
+from erpnext.accounts.utils import get_fiscal_year
+from frappe.utils import flt
+
+def getDateDiffDays(d1, d2):
+    return abs((d2 - d1).days)
+    
+def getDeprTillLastYear(day_before_start, fixed_asset_name):
+        for accdepr in frappe.get_doc("Fixed Asset Account", fixed_asset_name).depreciation:
+           if get_fiscal_year(fiscal_year = accdepr.fiscal_year) == get_fiscal_year(date = day_before_start):
+               return accdepr.total_accumulated_depreciation
+        return 0
+        
+def getDeprProvidedThisYear(method, depronopening, purchase_value, deprtilllastyr, rateofdepr):

```

Comment:
its very hard for someone to read `depronopening` should be `depreciation_at_opening`